homeassistant/components/zoneminder/__init__.py

```diff
@@ -1,6 +1,7 @@
 """Support for ZoneMinder."""
 import logging
 
+from requests.exceptions import SSLError
```

Comment:
```suggestion
from requests.exceptions import ConnectionError
```

Reply:
I've adjusted this as

`from requests.exceptions import ConnectionError as RequestsConnectionError`

Reply:
Importing as a different name is good.

Reply:
pylint

```
W0622: Redefining built-in 'ConnectionError' (redefined-builtin)
```